xbmc/utils/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ set(SOURCES ActorProtocol.cpp
             Crc32.cpp
             CSSUtils.cpp
             DatabaseUtils.cpp
+            DiscsUtils.cpp
             Digest.cpp
```

Comment:
Just for garbears sanity, Digest before DiscUtils for both cpp and h

Reply:
Nice catch